app/index.js

```diff
@@ -1,7 +1,14 @@
 import 'whatwg-fetch';
+<<<<<<< HEAD
```

Comment:
Looks like you committed merge conflicts. Remove this line that says `HEAD` all the way through the `=======`.